pkg/test/test_context.go

```diff
@@ -32,8 +34,9 @@ const prometheusOperatorVersion = "0.33"
 const certmanagerURL = "https://github.com/jetstack/cert-manager/releases/download/%s/cert-manager.yaml"
 const prometheusOperatorURL = "https://raw.githubusercontent.com/coreos/prometheus-operator/release-%s/bundle.yaml"
 
-// KBTestContext specified to run e2e tests
-type KBTestContext struct {
+//nolint:golint
+// TestContext specified to run e2e tests\
+type TestContext struct {
```

Comment:
TestContext is not a good name because it will be at the end test.TestContext. @estroz have you a better idea?
PS.: Context can not be used here.  (redeclared issue because of ginkgo import)